qiskit/circuit/instruction.py

```diff
@@ -417,22 +419,34 @@ def reverse_ops(self):
         reverse_inst.definition = reversed_definition
         return reverse_inst
 
-    def inverse(self):
+    def inverse(self, annotated: bool = False) -> Instruction | AnnotatedOperation:
```

Comment:
It's unfortunate that there are 2 `Instruction` classes in qiskit one for circuits and the other in pulse. The ci docs failure is because sphinx couldn't tell them apart with just the `Instruction` class listed here. Sphinx doesn't really understand type hinting in the same manner that python does apparently, because in python there is enough context to know that `Instruction` here is the class

Reply:
Thanks, I realized that before I saw your comment :).